### PR TITLE
Feature/phylop

### PR DESCRIPTION
**Pull request by rajatshuvro**

### CODE REVIEW

File: docs/data-sources/phylop.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Overview
+
+PhyloP (phylogenetic p-values) conservation scores are obtained from the [PHAST package] (http://compgen.bscb.cornell.edu/phast/) for multiple alignments of vertebrate genomes to the human genome. For GRCh38, the multiple alignments are against 19 mammals and for GRCh37, it is against 45 vertebrate genomes.
+
+:::info Publication
+Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, Clawson H, Spieth J, Hillier LW, Richards S, et al. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. **Genome Res. 2005** Aug;15(8):1034-50.  (http://www.genome.org/cgi/doi/10.1101/gr.3715005)
+:::
+
+
+## File Parsing
```
Comment (MichaelStromberg) [inline docs/data-sources/phylop.mdx]: "File Parsing" should be "Wigfix File" to be consistent with dbSNP.

File: docs/data-sources/phylop.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+0.111
+0.111
+0.100
+0.111
+-1.636
+```
+We convert them to binary files with indexes for fast query. Note that these are scores for genomic positions and are reported only for SNVs.
+## Download URL
+
+[GRCh37](http://hgdownload.cse.ucsc.edu/goldenpath/hg19/phyloP46way/vertebrate/)
```
Comment (MichaelStromberg) [inline docs/data-sources/phylop.mdx]: Go ahead and output the entire URL rather than labeling them GRCh37 and GRCh38. Similar to dbSNP.

**Review by MichaelStromberg — CHANGES_REQUESTED** [review]

File: docs/data-sources/phylop.mdx (hunk 1 above)
Comment (rajatshuvro) [reply docs/data-sources/phylop.mdx]: Done too.

File: docs/data-sources/phylop-json.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      "refAllele":"G",
+      "altAllele":"A",
+      "variantType":"SNV",
+      "phylopScore":0.459
+  }
+]  
+```
+| Field            | Type         | Notes                     |
+|:-----------------|:------------:|:--------------------------|
+| phylopScore      | float        | Range: -14.08 to 6.424    |
```
Comment (MichaelStromberg) [inline docs/data-sources/phylop-json.md]: Make this range to be consistent with the other data sources.

**Review by MichaelStromberg — CHANGES_REQUESTED** [review]

**Review by MichaelStromberg — APPROVED** [review]

**Pull request merged**